src/content/1.7/basics/keeping-up-to-date/migration.md

```diff
@@ -406,7 +406,7 @@ to change as shown on the screenshot.
 * Start by selecting your CSV file.
 * The second option to modify will be the format. Choose “CSV” or “CSV for MS Excel”.
 * Skip the first line of your file. We had the column names in the first line to make our file edition easier, but we do not want to import it in the database.
-* Disable the foreign key checks. We know some relations don’t exist yet, because they won’t be imported at the same time.
+* Disable the foreign key checks. We know some relations doesn’t exist yet, because they won’t be imported at the same time.
```

Comment:
```suggestion
* Disable the foreign key checks. We know some relations do not exist yet, because they won’t be imported at the same time.
```
Some relations is plural so `do not` instead of `does not`

Reply:
You're right